src/test/java/com/aws/greengrass/clientdevices/auth/iot/ThingRegistryTest.java

```diff
@@ -83,28 +67,4 @@ void GIVEN_unmodifiedThing_WHEN_updateThing_THEN_oldThingReturned() {
 
         // TODO: no update event
     }
-
-    @Test
```

Comment:
Removed this tests given they where testing a that bad api. I will write some integration tests to cover this